### DISCUSSION TITLE
1.1.1r Not Available in GitHub

### DISCUSSION THREAD

**Discussion by rsbeckerca** [openssl/openssl#23124] [Q&A] [answered]
I am trying to find the 1.1.1r release that just came out, but it does not seem to be in GitHub. Any pointers to the commit?

**Comment by t-j-h** [comment]
You will not find a download for that release (it is of course in github) as there was an issue and it was withdrawn.

https://www.openssl.org/news/newslog.html

> 12-Oct-2022 OpenSSL 3.0.6 and 1.1.1r are withdrawn. New releases will be created in due course.

**Comment by mattcaswell** [comment, marked as answer]
> 1.1.1r release that just came out,

There was no new release....1.1.1r was released (and withdrawn) over a year ago. A spurious email seems to have gone out that was a year old.
